websites/K/KOLEO/presence.ts

```diff
@@ -0,0 +1,980 @@
+const presence = new Presence({
+		clientId: "1265368122689458378",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/idEEfM8.png",
+	Logo2 = "https://i.imgur.com/mYXLmgF.png",
+	Buy = "https://i.imgur.com/Z8xQXY4.png",
+	Train = "https://i.imgur.com/Jz7uUX1.png",
+	Ticket = "https://i.imgur.com/PDNRKZ0.png",
+}
+
+let oldStations: string | string[];
+
+function NoPage(presenceData: PresenceData): void {
+	presenceData.name = "KOLEO - 404";
+	presenceData.details = "Nie znaleziono strony.";
+	presenceData.largeImageKey = Assets.Logo;
+	presenceData.state = "";
+	presenceData.smallImageText = "Zgubił się...";
+	presenceData.smallImageKey = Assets.Question;
+}
+
+const operators = [
+	"pkp-intercity",
+	"polregio",
+	"arriva",
+	"leo-express",
+	"koleje-wielkopolskie",
+	"koleje-dolnoslaskie",
+	"koleje-mazowieckie",
+	"koleje-malopolskie",
+	"wkd",
+	"lka",
+	"koleje-slaskie",
+	"skm-trojmiasto",
+];
+
+presence.on("UpdateData", async () => {
+	const { href } = document.location,
+		presenceData: PresenceData = {
+			details: "Ładowanie.",
+			largeImageKey: Assets.Logo,
+			smallImageText: "Ładowanie...",
+			smallImageKey: Assets.Train,
+			startTimestamp: browsingTimestamp,
+		},
+		[privacySetting] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+		]);
```

Comment:
Since it's only one setting use this.
```suggestion
		privacySetting = await presence.getSetting<boolean>("privacy");
```